css/style.css

```
@@ -17,3 +17,119 @@ body {
  */
 
 
+* {
+    font-weight: lighter;
+}
+
+/* Navigation styles */
+.topnav img {
+    margin-left: 4rem;
+}
+
+.topnav {
+    display:flex;
+    justify-content: space-between;
+}
+
+ul {
+    list-style-type: none;
+    padding: 0.5rem;
+    margin-right: 4rem;
+}
+
+li {
+    margin-left: 1.5rem;
+}
+
+li:first-of-type {
+    font-weight:700;
+}
+
+.nav-links {
+    display: flex;
+    justify-content: space-between;
+} 
+
+a:link, a:visited {
```

```suggestion
.topnav a {
```
It looks like we can apply this style to all `a` elements, so we can leave out `:link` and `:visited`. We can also scope these styles to only the topnav section by changing the selector to `.topnav a`, in case there are other `a` elements on the page

---

@bonboh thanks so much for your comments! I'll add some media queries and go back through and implement your suggestions, i definitely knew there would be some extra/redundant stuff in there :/